Lib/test/datetimetester.py

```diff
@@ -19,7 +19,7 @@
 from operator import lt, le, gt, ge, eq, ne, truediv, floordiv, mod
 
 from test import support
-from test.support import is_resource_enabled, ALWAYS_EQ, LARGEST, SMALLEST
+from test.support import import_fresh_module, is_resource_enabled, ALWAYS_EQ, LARGEST, SMALLEST
```

Comment:
This section is also going over the 79 char limit. Please adjust it accordingly.

Reply:
For this I'll go with parenthesis notation from the [PEP328](https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses) rather than backslashes.

Reply:
I think you won't need to change this line at all once you have dropped `test_c_all`.